services/wallet/price_test.go

```diff
@@ -0,0 +1,72 @@
+package wallet
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/status-im/status-go/appdatabase"
+	"github.com/status-im/status-go/sqlite"
+)
+
+func setupTestPriceDB(t *testing.T) (*PriceManager, func()) {
+	tmpfile, err := ioutil.TempFile("", "wallet-price-tests-")
```

Comment:
DB in memory should be enough and faster in memory for this type of test. See an example [here](https://github.com/status-im/status-go/blob/da73e8161273fafe248c539d2f6b332c872cde87/services/wallet/history/balance_db_test.go#L17)

Reply:
Thanks! I was just following the SavedAddresses example